pop-0001.txt

```diff
@@ -0,0 +1,623 @@
+*something* to be done, but there are details that need to be worked out
+before the POP can be accepted.
+
+If the final decision on a POP is to be made by a delegate rather than
+directly by the BDFL, this will be recorded by including the
+"BDFL-Delegate" header in the POP.
+
+POP review and resolution may also occur on a list other than PyLadies-dev
+(for example, distutils-sig for packaging related POPs that don't
+immediately affect the standard library). In this case, the "Discussions-To"
+heading in the POP will identify the appropriate alternative list where
+discussion, review and pronouncement on the POP will occur.
+
+For a POP to be accepted it must meet certain minimum criteria.  It
+must be a clear and complete description of the proposed enhancement.
+The enhancement must represent a net improvement.  The proposed
+implementation, if applicable, must be solid and must not complicate
+the interpreter unduly.  Finally, a proposed enhancement must be
+"PyLadiesic" in order to be accepted by the BDFL.  (However, "PyLadiesic"
+is an imprecise term; it may be defined as whatever is acceptable to
+the BDFL.  This logic is intentionally circular.)  See POP 2 [2]_ for
+standard library module acceptance criteria.
+
+Once a POP has been accepted, the reference implementation must be
+completed.  When the reference implementation is complete and incorporated
+into the main source code repository, the status will be changed to "Final".
+
+A POP can also be assigned status "Deferred".  The POP author or an
+editor can assign the POP this status when no progress is being made
+on the POP.  Once a POP is deferred, a POP editor can re-assign it
+to draft status.
+
+A POP can also be "Rejected".  Perhaps after all is said and done it
+was not a good idea.  It is still important to have a record of this
+fact. The "Withdrawn" status is similar - it means that the POP author
+themselves has decided that the POP is actually a bad idea, or has
+accepted that a competing proposal is a better alternative.
+
+When a POP is Accepted, Rejected or Withdrawn, the POP should be updated
+accordingly. In addition to updating the status field, at the very least
+the Resolution header should be added with a link to the relevant post
+in the PyLadies-dev mailing list archives.
+
+POPs can also be superseded by a different POP, rendering the original
+obsolete.  This is intended for Informational POPs, where version 2 of
+an API can replace version 1.
+
+The possible paths of the status of POPs are as follows:
+
+.. image:: POP-0001-1.png
+
+Some Informational and Process POPs may also have a status of "Active"
+if they are never meant to be completed.  E.g. POP 1 (this POP).
+
+
+POP Maintenance
+---------------
+
+In general, Standards track POPs are no longer modified after they have
+reached the Final state. Once a POP has been completed, the Language and
+Standard Library References become the formal documentation of the expected
```

Comment:
"Once a POP has been completed, the guidelines, kit documentation, code, etc become the formal..."